extensions/grpc/codegen/src/main/java/io/quarkus/grpc/deployment/GrpcCodeGen.java

```diff
@@ -146,6 +146,24 @@ public boolean trigger(CodeGenContext context) throws CodeGenException {
         return false;
     }
 
+    private static void copySanitizedProtoFile(Path protoPath, Path outProtoPath) throws IOException {
+        try (var reader = Files.newBufferedReader(protoPath);
+                var writer = Files.newBufferedWriter(outProtoPath)) {
+
+            String line = reader.readLine();
+            while (line != null) {
+                // filter java_generic_services to avoid "Tried to write the same file twice"
+                // when set to true. Generic services are deprecated and replaced by classes generated by
+                // this plugin
+                if (!line.contains("java_generic_services")) {
+                    writer.write(line);
```

Comment:
What about adding a warning here, indicating to the user that we had to change the proto file? If we could reference a place indicating that `java_generic_services` is deprecated, that would be really awesome.

Reply:
You mean logger warn ?

Reply:
I add a logger info. To me, warning means the user should do something to fix it. But I believe that this issue will only happen for dependency the user do not own and have no control over it.
Doc now states that this option should not be used and is ignored for dependencies.